test/images/agnhost/Dockerfile

```diff
@@ -26,10 +30,14 @@ RUN apk --update add bind-tools curl netcat-openbsd iproute2 iperf bash && rm -r
   && ln -s /usr/bin/iperf /usr/local/bin/iperf \
   && ls -altrh /usr/local/bin/iperf
 
-# PORT 8080 needed by: netexec, nettest
+ADD https://github.com/coredns/coredns/releases/download/v1.5.0/coredns_1.5.0_linux_BASEARCH.tgz /coredns.tgz
```

Comment:
This CoreDNS version has a bug that required  to move to 1.6.2 instead of using 1.5.0 https://github.com/kubernetes/kubeadm/issues/1745

Reply:
xref https://github.com/kubernetes/kubernetes/blob/master/build/dependencies.yaml

this might get weird with kube-up and kubeadm and whatnot being out of sync.